pkg/reconciler/buildrun/resources/image_processing.go

```diff
@@ -20,6 +22,28 @@ const (
 	paramOutputDirectory         = "output-directory"
 )
 
+type VulnerablilityScanParams struct {
+	build.VulnerabilityScanOptions
+}
+
+var _ pflag.Value = &VulnerablilityScanParams{}
+
+func (v *VulnerablilityScanParams) Set(s string) error {
+	return json.Unmarshal([]byte(s), v)
+}
+
+func (v *VulnerablilityScanParams) String() string {
+	data, err := json.Marshal(*v)
+	if err != nil {
+		panic(err.Error())
+	}
+	return string(data)
+}
+
+func (v *VulnerablilityScanParams) Type() string {
+	return "vulnerability-scan-params"
+}
+
```

Comment:
We should move that into the respective CLI tool package.

Reply:
As discussed, it is being used in other packages as well. So can't move it to the respective CLI package.